extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -37,29 +39,46 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 	protected readonly loginEndpointUrl: string;
 	public readonly commonTenant: Tenant;
+	public readonly organizationsTenant: Tenant;
```

Comment:
nit: I wouldn't include an 's' in organizations unless you're meaning plural.  Possessive tense isn't communicated well in variable names.